lib/models/student.js

```diff
@@ -36,6 +36,12 @@ module.exports = class Student extends Model {
             qualification: Joi.number().integer().valid( ..._.values(CONSTANTS.studentDetails.qualification) ).allow(null),
             currentStatus: Joi.number().integer().valid( ..._.values(CONSTANTS.studentDetails.currentStatus) ).allow(null),
             schoolMedium: Joi.number().integer().valid( ..._.values(CONSTANTS.studentDetails.schoolMedium) ).allow(null),
+           
+            // additional details
```

Comment:
The comments should be something descriptive and not this. If I was you I would write the comment as `academic details of class 10 and 12.`